src/pages/SmartWeaveExplorer/Interaction/Interaction.vue

```diff
@@ -260,23 +268,24 @@ export default {
       limit: 15,
       selected: 'all',
       copiedDisplay: false,
-      copiedDisplayOwner: false,
       copiedContractIdDisplay: false,
+      copiedDisplayCreator: false,
+      copiedDisplayBundlerId: false,
```

Comment:
I'll merge it for now, although I still don't like how we are handling copying ids, we should make it more generic, but it's an extra task for the future when we'll have a solution for this one :)

Reply:
I agree with you, and I think I found a solution, while working on refactor. So, it should be 'fixed' soon.

Reply:
cool!